hls4ml/utils/serialization.py

```diff
@@ -0,0 +1,225 @@
+import json
+import os
+import sys
+import tarfile
+import tempfile
+from pathlib import Path
+
+import numpy as np
+
+from hls4ml.model.graph import HLSConfig, ModelGraph
+from hls4ml.model.optimizer import optimize_model
+
+from .._version import version
+
+
+def serialize_model(model, file_path):
```

Comment:
Could we add a short docstring to this function?  Could be a copy of the docs, but still nice to have since it's a user-facing function.

Reply:
Sure I'll do that